cmake/kokkos_tribits.cmake

```diff
@@ -148,15 +148,20 @@ ELSE()
   CMAKE_PARSE_ARGUMENTS(PARSE
     ""
     ""
-    "SOURCES;CATEGORIES"
+    "SOURCES;CATEGORIES;ARGS"
     ${ARGN})
   VERIFY_EMPTY(KOKKOS_ADD_EXECUTABLE_AND_TEST ${PARSE_UNPARSED_ARGUMENTS})
   KOKKOS_ADD_TEST_EXECUTABLE(${ROOT_NAME}
     SOURCES ${PARSE_SOURCES}
   )
+  IF(DEFINED PARSE_ARGS)
+    STRING(REPLACE ";" " " PARSE_ARGS_STR "${PARSE_ARGS}")
+    SET(OPTIONAL_ARGS CMD_ARGS "${PARSE_ARGS_STR}")
```

Comment:
Probably better to just match TRIBITS and call this "ARGS" instead of "CMD_ARGS"

Reply:
Not sure I understand.
Are you asking me to rename?https://github.com/kokkos/kokkos/blob/ab1309810c05513ec47b90584840ce0ca2a071ed/cmake/fake_tribits.cmake#L117

Reply:
Yes, it seemed confusing to call it "ARGS" in one place and "CMD_ARGS" in another. This is probably my fault from long ago.